agent_application/docker/docker-compose.yml

```diff
@@ -27,12 +27,16 @@ services:
     depends_on:
       - cqrs-postgres-db
   ssi-agent:
-    image: ssi-agent
-    restart: always
+    #image: ssi-agent
```

Comment:
Maybe we should rather introduce a separate `docker-compose.dev.yml` or something. We cannot expect someone who just wants to run the application to know where and what to uncomment.

Reply:
No we shouldn't in my opinion. If people want to have a production environment they have to setup themselves. Not just use ours. Thats also why I don't think we should restart, its just for us. Or they can use it as a template but nothing more than that.

Reply:
Okay, we could remove `#image: ssi-agent` then altogether, right?